CHANGELOG.next.toml

```
@@ -22,3 +22,14 @@ message = "Log a debug event when a retry is going to be peformed"
 references = ["smithy-rs#1352"]
 meta = { "breaking" = false, "tada" = false, "bug" = false }
 author = "jdisanti"
+
```

```suggestion

[[aws-sdk-rust]]
```
missing the header for this entry